### ISSUE TITLE
Improve site organization + visibility: Delete Prefix search page

### ISSUE THREAD

**Issue by maryamsulemani97** [meilisearch/documentation#2396]
As part of https://github.com/meilisearch/documentation/issues/2380, we would like to remove a few pages.

## Possible solutions

1. Instead of deleting the [Prefix search](https://www.meilisearch.com/docs/learn/advanced/prefix) page, we can create a section called "Types of search" (or something else) on the [search API reference](https://www.meilisearch.com/docs/reference/api/search) and add placeholder, phrase, and prefix search there. That way, we won't have to duplicate content for both GET and POST search routes.

### Cons

Goes against our API reference format

2. Create a section called Search (need a more helpful name) and move all search-related content there

### Cons

Not something we can do immediately, as we only have two short pages on search: Concatenate and split queries and Prefix search. Could be done after #1626

**Comment by dichotommy** [comment]
> 2. Create a section called Search and move all search-related content there

I'm not convinced that a section called search would be clear or helpful for users, since it is so broad—pretty much every part of Meilisearch can be related to search. But I think you might be on the right track. A title like "Understanding search results" could give a clearer focus. After, the question is whether it should be a section or a single page.

Our current problem is that we have these two short pages ("Prefix search" + "Concatenate and split queries") that push other more important pages down in the sidebar. "Tokenization" could be viewed as another one. 

It seems like rather than deleting the page, perhaps the best thing for us to do right now is actually to move the "Inner workings" section to the bottom of the sidebar, to increase the visibility of more practical articles.

**Comment by maryamsulemani97** [comment]
>I'm not convinced that a section called search would be clear or helpful for users, since it is so broad—pretty much every part of Meilisearch can be related to search. But I think you might be on the right track. A title like "Understanding search results" could give a clearer focus.

Agreed. Should have clarified this 😅 

>It seems like rather than deleting the page, we should move the entire "Inner workings" section towards the bottom of the sidebar, to increase the visibility of more practical articles.

We can do that. But it would be helpful to group the "types of searches"(placeholder, phrase, prefix) on one page/section. Since we don't have any other dedicated search guides, we can wait until after #1626

**Comment by curquiza** [comment]
Discussed with @guimachiavelli 
This issue is stale, and gui was not involved in this project, so not able to continue it.
Feel free to let us if we need to re-open a new issue 😊
